tests/unit/query/js-adapter-test.js

```diff
@@ -627,6 +627,24 @@ test('adapter | js | true predicate | detail predicate', (assert) => {
   assert.equal(result.length, 3);
 });
 
+test('adapter | js | true predicate | detail predicate', (assert) => {
```

Comment:
Для предыдущего теста можно тоже имя уточнить.
```suggestion
test('adapter | js | true predicate | detail predicate all', (assert) => {
```